### PR TITLE
feat(webapp): implement config for `date-time` format

### LINKED ISSUE #6979: 🚀 [Feature][Webapp] Configure if Relative Datetime or Absolut Datetime
## 🚀 Feature

Configure if Relative Datetime or Absolute Datetime.

### Todo

- [x] make relative date configurable (?) (Tuesday => 17.02)

### PR DESCRIPTION
**Pull request by Tirokk**
## 🍰 Pullrequest

Implement config for `date-time` format.

### Issues

- fixes #6979

### Todo

- [x] none

### CODE REVIEW

File: CONTRIBUTING.md
Comment (Tirokk) [inline CONTRIBUTING.md:1]: Sorry for this unrelated changes, but didn't like to open an other issue. 🙏🏼

File: README.md
Comment (Tirokk) [inline README.md:1]: Sorry for this unrelated changes, but didn't like to open an other issue. 🙏🏼

File: webapp/constants/dateTime.js
Comment (Tirokk) [inline webapp/constants/dateTime.js:1]: Got this as I did `yarn install` in the main folder.
Don't know why.

File: yarn.lock
Comment (Tirokk) [inline yarn.lock:1]: Got this as I did `yarn install` in the main folder.
Don't know why.

**Review by Elweyn — APPROVED** [review]

**Pull request merged**